adapters/LiftoffMonetize/LiftoffMonetizeAdapter/VungleRouterConsent.m

```diff
@@ -13,23 +13,20 @@
 // limitations under the License.
 
 #import "VungleRouterConsent.h"
-#import <VungleSDK/VungleSDK.h>
-
-static VungleConsentStatus currentConsentStatus = 0;
+#import <VungleAdsSDK/VungleAdsSDK.h>
 
 @implementation VungleRouterConsent
-+ (BOOL)updateConsentStatus:(VungleConsentStatus)consentStatus {
-  VungleSDK *sdk = VungleSDK.sharedSDK;
-  if (consentStatus > 0) {
-    currentConsentStatus = consentStatus;
-    [sdk updateConsentStatus:consentStatus consentMessageVersion:@""];
-    return YES;
-  }
-  return NO;
+
++ (void)updateGDPRStatus:(BOOL)optIn {
+  [VunglePrivacySettings setGDPRStatus:optIn];
+}
+
++ (void)updateCCPAStatus:(BOOL)optIn {
+  [VunglePrivacySettings setCCPAStatus:optIn];
 }
 
-+ (VungleConsentStatus)getConsentStatus {
-  return currentConsentStatus;
++ (void)updateIDFVStatus:(BOOL)optIn {
+  [VunglePrivacySettings setPublishIdfv:optIn];
```

Comment:
Is there a reason this class is still here? I think this class can be removed.
It looks like the publishers can directly call these `VunglePrivacySettings` methods statically and we can update the docs to clarify they need to be set before ad request time.

Reply:
@joshuajplagonera Another reason why it was wanted was to avoid less direct interaction with our SDK so if the API changes, the migration for the publisher would be easier.

I got the approval to remove this if this is a blocker for the certification.

Reply:
Thanks for the explanation. This can be kept. :)

Reply:
Ok. I'll remove the class.

The consent values should be set prior to the requesting ads of the Vungle SDK (through the GMA SDK). Currently our SDK does not utilize any identifiers for the initialization but it will be passed when the ad request is made.